tutorials/nservicebus-sagas/2-timeouts/Snippets/Snippets.sln.DotSettings

```diff
@@ -0,0 +1,6 @@
+<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
This entire file needs to be ignored and removed, not?

Reply:
Good question, it's also in lesson 1. I don't have R# so I don't need it, I have no idea why it's there.